packages/select/src/vaadin-select.js

```diff
@@ -627,6 +643,49 @@ class Select extends OverlayClassMixin(
     itemElement.setAttribute('id', this._itemId);
   }
 
+  /**
+   * @param {string} accessibleName
+   * @private
```

Comment:
Should this be `@protected` as it's inherited? 
Same actually applies to this method in the mixin.

Reply:
Done.